modules/test-structure/save_test_data.go

```diff
@@ -282,3 +282,30 @@ func CleanupTestData(t *testing.T, path string) {
 		logger.Logf(t, "%s does not exist. Nothing to cleanup.", path)
 	}
 }
+
+// CleanupTestData cleans up the .test-data folder inside the given folder.
```

Comment:
The first line of a comment preceding a function should generally begin with exactly the function name. Looks like that was the intent here but perhaps the function name changed? In any case, I'd start with `CleanupTestDataFolder cleans up the...`.